Sources/SwiftDocC/Infrastructure/Link Resolution/ExternalPathHierarchyResolver.swift

```diff
@@ -165,7 +165,7 @@ extension LinkDestinationSummary {
     
     /// Create a topic render render reference for this link summary and its content variants.
     func makeTopicRenderReference() -> TopicRenderReference {
-        let (kind, role) = DocumentationContentRenderer.renderKindAndRole(kind, semantic: nil)
+        let (kind, role) = DocumentationContentRenderer.renderKindAndRole(kind, semantic: nil, linkSummary: self)
```

Comment:
```suggestion
        let (kind, role) = DocumentationContentRenderer.renderKindAndRole(kind, linkSummary: self)
```

You set the default value for `semantic` to be nil, so there's no need to pass that in here now.